dev/get_schema.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Generate latest schema file for reference in development 
+# To be run from project root (so, './dev/get_schema.sh')
```

Comment:
Can you make it work from any directory? You can find out where the script is located and then point the schema to `$PWD/../apps/`, similarly to how it's done here:

https://github.com/mediacloud/backend/blob/d1da1bebba9430693ee9a32d0daa3f717b53dbb1/apps/nytlabels-annotator/src/crappy-predict-news-labels/nytlabels.sh#L6-L8